ExpressionUtils/LambdaParts.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+
+namespace MiaPlaza.ExpressionUtils {
+
+	/// <summary>
+	/// Sometimes we don't have actual <see cref="LambdaExpression"/> object, but instead <see cref="LambdaExpression.Body"/> and <see cref="LambdaExpression.Parameters"/> that represent lambda.
+	/// We could call <see cref="Expression.Lambda(Expression, ParameterExpression[])"/> to get it, but such calls can be expensive due to locking that is used inside.
+	/// So this object plays a role of a container that holds the data that is needed to create lambda expression.
```

Comment:
```suggestion
	/// So this object plays a role of a container that holds the data that is needed to represent a lambda expression.
```

(as we want to avoid creating the lambda expression)